### DISCUSSION TITLE
resource argoproj.io:AppProject is not permitted in project xxx

### DISCUSSION THREAD

**Discussion by Vanderscycle** [argoproj/argo-cd#16512] [Q&A]
given an application managing all of the projects:
```yaml
apiVersion: argoproj.io/v1alpha1
kind: Application
metadata:
  name: application-projects
  namespace: argocd
spec:
  destination:
    namespace: argocd
    server: https://kubernetes.default.svc
  project: applications
  source:
    path: argocd/projects
    repoURL: git@github.com:Vanderscycle/infrastructure.git
    targetRevision: main
  syncPolicy:
    automated:
      selfHeal: true
      prune: false
```
and, as an example, here's a very permissive project:
```yaml
apiVersion: argoproj.io/v1alpha1
kind: AppProject
metadata:
  name: applications
  namespace: argocd
  finalizers:
  - resources-finalizer.argocd.argoproj.io
spec:
  description: Project for parent applicatons used when employing app of apps
  destinations:
    - namespace: '*'
      server: '*'
    # - namespace: argocd
    #   server: https://kubernetes.default.svc
    # - namespace: kube-system
    #   server: https://kubernetes.default.svc
  clusterResourceWhitelist:
    - group: "*"
      kind: "*"
    # - group: 'argoproj.io'
    #   kind: 'AppProject'
  namespaceResourceWhitelist:
    - group: "*"
      kind: "*"
    # - group: "argoproj.io"
    #   kind: Application
  sourceRepos:
    - "*"
  sourceNamespaces:
    - "*"
```

I can't seem to sync at all and it was working in the past with no issues.
![image](https://github.com/argoproj/argo-cd/assets/52706578/c25b667f-984b-43b5-8d1b-34c7a8206e2c)

This is weird, because I should have the required permissions to be present but it isn't allowed.

I have read and tried the following:
-  https://github.com/argoproj/argo-cd/issues/11597
- https://github.com/argoproj/argo-cd/issues/3913

other project looks as such:
```yaml
apiVersion: argoproj.io/v1alpha1
kind: AppProject
metadata:
  name: servers
  namespace: argocd
  finalizers:
    - resources-finalizer.argocd.argoproj.io
spec:
  description: A game server project
  destinations:
    - namespace: "*"
      server: "*"
    # - namespace: factorio
    #   server: https://kubernetes.default.svc
    # - namespace: factorio-space
    #   server: https://kubernetes.default.svc
    # - namespace: factorio-vanilla
    #   server: https://kubernetes.default.svc
    # - namespace: kube-system
    #   server: https://kubernetes.default.svc
  clusterResourceWhitelist:
    - group: "*"
      kind: "*"
  sourceRepos:
    - "*"
    # - git@github.com:Vanderscycle/infrastructure.git
```
Any help is appreciated

**Comment by Vanderscycle** [comment]
tldr; fixed through the argocd ui in the project tabs.
